src/lib/jsonrpc.ml

```diff
@@ -43,6 +43,7 @@ let rec rpc_to_json t =
   | Float r -> `Float r
   | String s -> `String s
   | DateTime d -> `String d
+  | Base64 b -> `String b
```

Comment:
It’s really annoying that in json we don’t have a way to infer the type. Please add a comment on `json_to_rpc` as we have for the DateTime